### PR TITLE
i have made all the recommended changes.Mistakenly, i had closed the previous pull  request.

### PR DESCRIPTION
**Pull request by bhavesh0902**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,188 @@
-# Fetching-Bot-Rooh-
-Srishti 2020
+# **Fetching-bot-Rooh**
+---
+## Srishti 2020
```
Comment (SanjeevKrishnan) [inline README.md]: Make "Srishti 2020" as normal text, not as heading.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,188 @@
...
+
+
+![Final Product](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20and%20Videos/Images/bot.jpg)
+
+---
+
+## Motivation
+It can be used for general household purposes and even in distribution chain of any product, thus making our job easy.
+
+![Final Product](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20and%20Videos/Images/Flowchart%20final.jpg)
```
Comment (SanjeevKrishnan) [inline README.md:23]: Flowchart has to come under second-level heading "Workflow"

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@bhavesh0902 Minor Changes
1. You were asked to add "Models and Robotics Section", and its logo in Report, not in README.md. Remove it from README.md and add it in Report.
2. Try to use cropped images in README.md Your screenshot has other unnecessary elements which is degrading the quality of your README file.

**Comment by bhavesh0902** [comment]
@SanjeevKrishnan  
Sir, I have made all the recommended changes.

**Review by SanjeevKrishnan — APPROVED** [review]
@bhavesh0902 PR looks fine!
@Shubhanshu07 I suggest for merge.

**Comment by agshubhs** [comment]
@bhavesh0902 PR looks good to  me also

**Pull request merged**